webapp/components/PollutionMapSearchBox.tsx

```
@@ -0,0 +1,155 @@
+
+function HilightLabel(props:{HilightText:string, value:string}) {
+  if (!props?.value || !props?.HilightText) {
+    return <span>props?.value</span>;
+  }
+  const text: string = props.value;
+  const subString = props?.HilightText
+    ? props.HilightText?.toLowerCase().substring(0, text.length)
+    : "";
+  const startIdx = text.toLowerCase().indexOf(subString);
+  if (startIdx == -1) {
+    return <Fragment>{text}</Fragment>;
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)